remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/logging/Log.java

```diff
@@ -58,4 +58,7 @@
          " 'option indexed_by_default = false;' to your schema file to disable indexing of message types" +
          " that do not have indexing annotations.", id = 28012)
    void legacyIndexingIsDeprecated(String typeName, String fileName);
+
+   @Message(value = "Error initializing the serialization context", id = 28013)
```

Comment:
The error message is not helpful enough for the average Joe. Let's say ```Error during execution of protostream serialization context initializer %s in module %```, where first arg is the class name and second arg is module name/id.

Reply:
Hmmm, maybe module name is hard to obtain, and the class name is already in the stacktrace. Still, let's change the error message.